packages/nimble-components/src/table/specs/table-column-specs/table-column-formatted-text.md

```diff
@@ -161,22 +161,32 @@ Follows the [Column Type Philosophy](/packages/nimble-components/src/table/specs
 -   `format` - a string which controls how the number is formatted for display. It can take one of the following values:
     -   `undefined` - use a default formatter that will display integers with no trailing zeros, limits to 6 digits, and exponential notation is used for numbers that are large (\`>= 1e6\`) or small (\`< 1e-3\`) in magnitude. Will be displayed left-aligned by default (since numbers will display an inconsistent number of fractional digits).
-    -   `'decimal'` - format all values as decimal values (e.g. 123.45), always displaying `decimal-digits` digits after the separator and never displaying exponential notation. If required, values will be rounded to reach the specified number of decimial digits. Configuring `decimal-digits` to `0` will round the value to the nearest integer and display it with no decimal places. Will be displayed right-aligned by default.
+    -   `'decimal'` - format all values as decimal values (e.g. 123.45) with a number of digits after the separator determined by `decimal-digits` or `decimal-maximum-digits`. Exponential notation is never used. If required, values will be rounded to reach the specified number of decimial digits. Configuring `decimal-digits` to `0` will round the value to the nearest integer and display it with no decimal places. Will be displayed right-aligned by default.
     -   This could be extended to other pre-configured formats in future. Their configuration attributes would be prefixed with the name of the format mode.
     -   **Note:** all of the above will be implemented using a `Intl.NumberFormat` formatter. Nimble will configure the formatter with defaults to match the [visual design spec](https://github.com/ni/nimble/issues/887). The exception is that we will set `useGrouping: true` to achieve `1,000` rather than `1000` because this styles the values in a way that is more human readable.
--   `decimal-digits` - when format is `decimal`, a number that controls how many digits are shown to the right of the decimal separator. Defaults to 2 if unspecified. Formats other than `decimal` ignore `decimal-digits`.
+-   `decimal-digits` - when format is `decimal`, a number that controls how many digits are shown to the right of the decimal separator. Defaults to 2 unless `decimal-maximum-digits` is specified, in which case a number of digits between 0 and `decimal-maximum-digits` are shown. Formats other than `decimal` ignore `decimal-digits`.
+-   `decimal-maximum-digits` - when format is `decimal`, a number that controls the maximum number of digits shown to the right of the decimal separator. Possible values are from 0 to 20. Defaults to 3. If `decimal-digits` is also specified, exactly that number of digits are shown and `decimal-maximum-digits` is ignored. Formats other than `decimal` ignore `decimal-maximum-digits`.
```

Comment:
Using `decimal-maximum-digits` and/or `unit-family` (specifically `unit-family` when not all values result in the same unit) will mess up the right-alignment that gets applied automatically with the `decimal` format. We probably need to update the logic for the `default` alignment to account for this.

Reply:
That makes sense.